src/main/java/frc/utils/applicationsutils/FileCreator.java

```diff
@@ -0,0 +1,85 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.io.File;
+import java.io.FileWriter;
+
+public class FileCreator {
+
+    public static final File OUTPUT_FILE = DirectoryPathsConstants.OUTPUT_FILES_DIRECTORY_PATH.resolve("FileCreator.txt").toFile();
+
+    public static File createFile(File file) {
+        try {
+            file.createNewFile();
+            return file;
+        }
+        catch (Exception exception) {
+            writeToOutputFile(OUTPUT_FILE, "\nException while creating file " + file + "\n" + exception, true);
+            return null;
+        }
+    }
+
+    public static void writeToTextFile(File file, String text, boolean isAppending) {
```

Comment:
If you don't ever use it, why it is here? You can delete it OR do something like:
try{ 
    write();
} catch (exception){
     try { writeToFile(exception);} 
     catch (exception) {printStackTrace();}
}